buildenv/jenkins/JenkinsfileBase

```diff
@@ -512,6 +512,17 @@ def setup() {
 		VENDOR_TEST_DIRS = (params.VENDOR_TEST_DIRS) ? "--vendor_dirs \"${params.VENDOR_TEST_DIRS}\"" : ""
 		VENDOR_TEST_SHAS = (params.VENDOR_TEST_SHAS) ? "--vendor_shas \"${params.VENDOR_TEST_SHAS}\"" : ""
 
+		def vendorRepos = params.VENDOR_TEST_REPOS.split(',')
+		// Check if any vendor repo contains SVTTestRepo
+		for (int i = 0; i < vendorRepos.size(); i++) {
+			def repoURL = vendorRepos[i]
+			echo "Vendor repo is ${repoURL}"
+			if (repoURL.contains('SVTTestRepo')) {
+				env.IS_SVT_TESTREPO = 'true'
+				echo "IS_SVT_TESTREPO is set to ${env.IS_SVT_TESTREPO}"
+			}
+		}
+
 		// handle three cases (true/false/null) in params.TEST_IMAGES_REQUIRED and params.DEBUG_IMAGES_REQUIRED
```

Comment:
Could the value be set without `split`? e.g. `params.VENDOR_TEST_REPOS.contains('SVTTestRepo')`.
Also, could this setting be merged together with the initial env.IS_SVT_TESTREPO setting with `= ? :`?